### PR TITLE
Modified signup Artist

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Ok for me

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -16,8 +16,11 @@ function App() {
         <Route path="/profile" element={<Profile />} />
         <Route path="/profile/:profileId" element={<Profile />} />
         <Route path="/register" element={<Register />} />
-        <Route path="/register/artist" element={<SignupArtist />} />  
-        <Route path="/register/artist/validation" element={<RegisterArtist />} />
+        <Route path="/register/artist" element={<SignupArtist />} />
```
Comment (LuigiMenez) [inline frontend/src/App.jsx:19]: Why do I feel like I have a file in a PR, you haven't left the dev?

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/App.jsx (hunk 1 above)
Comment (LoicBrassart) [reply frontend/src/App.jsx:19]: That's because they didn't: I've read the same three lines of code for 3 PRs in the last five minutes

**Review by LoicBrassart — APPROVED** [review]
Still not a good idea to create branches from anything but `dev`

**Pull request merged**